Source/WebKit/WebProcess/Extensions/Bindings/JSWebExtensionWrapper.h

```diff
@@ -29,7 +29,11 @@
 #include "WebFrame.h"
 #include "WebPage.h"
 #include <JavaScriptCore/JSRetainPtr.h>
+#if PLATFORM(COCOA)
 #include <JavaScriptCore/JavaScriptCore.h>
+#else
+#include <JavaScriptCore/JavaScript.h>
+#endif
```

Comment:
```suggestion
#include <JavaScriptCore/JavaScriptCore.h>
```

Reply:
I'm not sure if something isn't right in the WebKit build config, but JavaScriptCore.h doesn't exist on my build, but JavaScript.h does